plugins/mcreator-localization/help/fr_FR/dimension/portal_igniter_texture.md

```diff
@@ -1 +1 @@
-The texture of the portal igniter used to create the portal from the portal frame.
\ No newline at end of file
+La texture de l'allumeur du portail utilisé pour créer le portail avec le cadre du portail.
```

Comment:
Allumeur -> Briquet

Reply:
No because a lighter is not always used to create a portal. You could use powder or anything.